themes/geekboot/layouts/partials/survey.html

```diff
@@ -0,0 +1,5 @@
+<div class="docs-survey card mb-3 text-center col-10 mx-auto">
+    <div class="card-body fs-4">
+        Click to <a href="https://s.surveyplanet.com/uiuwwdxb">take the Crossplane user survey</a>. <br />Survey closes December 31.
```

Comment:
Is there any way we could also promote the "Crossplane Community Annual Report Survey" located at https://tinyurl.com/3b5cyfap?

That would be really helpful to get more data there, and potentially avoid confusion with the multiple surveys happening right now.  One way to do that may be to make this more clear that it's a documentation focused survey.

Reply:
For sure! When do you plant to close the annual report survey?

Reply:
I take this back - I think i'd rather focus our graduation efforts around a public ADOPTERS.md file, instead of this survey.  Your docs feedback survey is timeboxed, let's just focus on that for now :)